silk/views/request_detail.py

```diff
@@ -40,3 +40,13 @@ def get(self, request, request_id):
             'request': request
         }
         return render(request, 'silk/request.html', context)
+
+    @method_decorator(login_possibly_required)
+    @method_decorator(permissions_possibly_required)
+    def post(self, request, request_id):
+        silk_request = Request.objects.get(pk=request_id)
+        try:
+            silk_request.delete()
+        except Request.DoesNotExist:
+            return HttpResponseRedirect(request.headers.get('Referer', '/silk'))
+        return redirect('/silk')
```

Comment:
Probably better to call `reverse("summary")` here and the line above as well to not hardcode `/silk` path 
(Not everyone will use the `/silk` in their paths)

Reply:
Thanks for the feedback. I've now removed the hard-coded path and replaced it with `reverse('silk:summary')`.